code/game/objects/effects/effect_system/foam.dm

```diff
@@ -144,7 +144,7 @@
 	else
 		src.location = get_turf(location)
 
-	src.spread_amount = round(sqrt(spread_amount / 3), 1)
+	src.spread_amount = round(sqrt(spread_amount / 3))
 	carry.copy_to(carrying_reagents, carry.total_volume)
 	src.foam_flags = foam_flags
```

Comment:
use FLOOR?

Reply:
How?

Couldn't find it in the documentation?

Reply:
Floor does exactly the same thing tivi. And it's not in the doc jpr, it's a custom macro